ui/app/routes/optimize/summary.js

```diff
@@ -0,0 +1,29 @@
+import Route from '@ember/routing/route';
+import notifyError from 'nomad-ui/utils/notify-error';
+
+export default class OptimizeSummaryRoute extends Route {
+  breadcrumbs(model) {
+    if (!model) return [];
+
+    return [
+      {
+        label: model.slug.replace('/', ' / '),
```

Comment:
I thought for a second that this might make it look like two separate breadcrumbs, but I checked it out and I think it's good!